### PR TITLE
reference/developer-guides/sdk-modifying-flatcar: Host system requirements for arm64

### PR DESCRIPTION
**Pull request by pothos**
The arm64 target requires binary translation to be set up on the host system.
Document how to this can be set up, tested, and what the outcomes are.

### CODE REVIEW

File: docs/reference/developer-guides/sdk-modifying-flatcar.md [hunk 1]
```diff
@@ -162,15 +163,40 @@ The `--board` option can be set to one of a few known target architectures, or s
 To create a root filesystem for the `amd64-usr` target beneath the directory `/build/amd64-usr/`:
 
 ```shell
-./setup_board --default --board=amd64-usr
+./setup_board [--default] --board=amd64-usr
 ```
 
+##### 64 bit ARM: The arm64-usr target
+
+The SDK assumes to run on an amd64 host system, relying on cross-compilation to create arm64 binaries.
```
Comment (marga-kinvolk) [inline docs/reference/developer-guides/sdk-modifying-flatcar.md:171]: "assumes to run" is broken grammar, but I'm not sure what you're trying to say. Maybe "The SDK is expected to be run on ..."? Or maybe "SDK tools assume they are run on ..."

**Review by marga-kinvolk — APPROVED** [review]

File: docs/reference/developer-guides/sdk-modifying-flatcar.md (hunk 1 above)
Comment (pothos) [reply docs/reference/developer-guides/sdk-modifying-flatcar.md:171]: :smile: I think making it shorter would work: _The SDK runs on ..._

Comment (pothos) [reply docs/reference/developer-guides/sdk-modifying-flatcar.md]: Changed it now

Comment (vbatts) [reply docs/reference/developer-guides/sdk-modifying-flatcar.md]: i've meant to open an issue for how to bootstrap an arm64 host sdk...

**Pull request merged**